src/pretix/plugins/sendmail/views.py

```diff
@@ -40,7 +42,16 @@ def form_valid(self, form):
         for o in orders:
             try:
                 mail(o.email, form.cleaned_data['subject'], form.cleaned_data['message'],
-                     None, self.request.event, locale=o.locale, order=o)
+                     {
+                         'event': o.event,
+                         'order': o.code,
+                         'order_date': date_format(o.datetime, 'SHORT_DATETIME_FORMAT'),
+                         'due_date': date_format(o.expires, 'SHORT_DATE_FORMAT'),
+                         'order_url': build_absolute_uri(o.event, 'presale:event.order', kwargs={
+                             'order': o.code,
+                             'secret': o.secret
+                         })
+                }, self.request.event, locale=o.locale, order=o)
```

Comment:
The indentation of this and the above } should match the line of the opening one :)